TradeItIosEmsApi/TradeItPosition.h

```diff
@@ -46,4 +46,6 @@
 
 @property (nullable, copy) NSString<Optional> *currencyCode;
 
+@property (nonnull, copy) NSString  *positionDescription;
```

Comment:
Is this always `nonnull`?

Reply:
It is set to empty string in the server by default:
`String description = ""`
It could be null if the broker returns null but in that case we should return the symbol (It is the case for IB). 
I could change it to nullable to be sure.

Reply:
Yeah let's change it to nullable for now - I feel like some broker implementations won't have it.